src/arrayMethodPop.js

```diff
@@ -5,7 +5,13 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    if (this.length < 1) {
```

Comment:
отрицательная длина не может быть
```suggestion
    if (this.length === 0) {
```
```suggestion
    if (!this.length) {
```